packages/base/src/UI5Element.js

```diff
@@ -925,6 +925,10 @@ class UI5Element extends HTMLElement {
 		return "";
 	}
 
+	static get _getElementsWithMultipleTags() {
```

Comment:
We have this.getMetadata().getTag(), this _getElementsWithMultipleTags in UI5Element look very strange to me. I would go for a implementation at metadata level.